components/resources/commands.grdp

```diff
@@ -0,0 +1,472 @@
+    Bookmark bar new folder
+  </message>
+  <message name="IDS_IDC_BOOKMARK_MANAGER">
+    Bookmark manager
+  </message>
+  <message name="IDS_IDC_BOOKMARK_BAR_ALWAYS_SHOW">
+    Bookmark bar always show
+  </message>
+  <message name="IDS_IDC_BOOKMARK_BAR_SHOW_APPS_SHORTCUT">
+    Bookmark bar show apps shortcut
+  </message>
+  <message name="IDS_IDC_BOOKMARK_BAR_SHOW_READING_LIST">
+    Bookmark bar show reading list
+  </message>
+  <message name="IDS_IDC_BOOKMARK_BAR_SHOW_MANAGED_BOOKMARKS">
+    Bookmark bar show managed bookmarks
+  </message>
+  <message name="IDS_IDC_BOOKMARK_BAR_ADD_TO_BOOKMARKS_BAR">
+    Bookmark bar add to bookmarks bar
+  </message>
+  <message name="IDS_IDC_BOOKMARK_BAR_REMOVE_FROM_BOOKMARKS_BAR">
+    Bookmark bar remove from bookmarks bar
+  </message>
+  <message name="IDS_IDC_CONTENT_CLIPBOARD_HISTORY_MENU">
+    Content clipboard history menu
+  </message>
+  <message name="IDS_IDC_MEDIA_ROUTER_SHOW_IN_TOOLBAR">
+    Show media router in toolbar
+  </message>
+  <message name="IDS_IDC_MEDIA_STREAM_DEVICE_STATUS_TRAY">
+    Media stream device status tray
+  </message>
+  <message name="IDS_IDC_MEDIA_STREAM_DEVICE_ALWAYS_ALLOW">
+    Media stream device always allow
+  </message>
+  <message name="IDS_IDC_TAB_SEARCH">
+    Tab search
+  </message>
+  <message name="IDS_IDC_TAB_SEARCH_CLOSE">
+    Tab search close
+  </message>
+  <message name="IDS_IDC_DEBUG_TOGGLE_TABLET_MODE">
+    Debug toggle tablet mode
+  </message>
+  <message name="IDS_IDC_DEBUG_PRINT_VIEW_TREE">
+    Debug print view tree
+  </message>
+  <message name="IDS_IDC_DEBUG_PRINT_VIEW_TREE_DETAILS">
+    Debug print view tree details
+  </message>
+  <message name="IDS_IDC_LIVE_CAPTION">
+    Toggle live captions
+  </message>
+  <message name="IDS_IDC_RUN_SCREEN_AI_VISUAL_ANNOTATIONS">
+    Run screen AI visual annotations
+  </message>
+  <message name="IDS_IDC_SIDEBAR_SHOW_OPTION_ALWAYS">
+    Show sidebar
+  </message>
+  <message name="IDS_IDC_SIDEBAR_SHOW_OPTION_MOUSEOVER">
+    Show sidebar on mouseover
+  </message>
+  <message name="IDS_IDC_SIDEBAR_SHOW_OPTION_NEVER">
+    Hide sidebar
+  </message>
+  <message name="IDS_IDC_SHOW_BRAVE_WALLET_PANEL">
+    Show brave wallet panel
+  </message>
+  <message name="IDS_IDC_CLOSE_BRAVE_WALLET_PANEL">
+    Close brave wallet panel
+  </message>
+  <message name="IDS_IDC_SHOW_BRAVE_VPN_PANEL">
+    Show brave VPN panel
+  </message>
+  <message name="IDS_IDC_BRAVE_VPN_MENU">
+    Brave VPN menu
+  </message>
+  <message name="IDS_IDC_TOGGLE_BRAVE_VPN_TOOLBAR_BUTTON">
+    Toggle brave VPN toolbar button
+  </message>
+  <message name="IDS_IDC_SEND_BRAVE_VPN_FEEDBACK">
+    Send brave VPN feedback
+  </message>
+  <message name="IDS_IDC_ABOUT_BRAVE_VPN">
+    About brave VPN
+  </message>
+  <message name="IDS_IDC_MANAGE_BRAVE_VPN_PLAN">
+    Manage brave VPN plan
+  </message>
+  <message name="IDS_IDC_TOGGLE_BRAVE_VPN">
+    Toggle brave VPN
+  </message>
```

Comment:
Brave should be capitalized in these.

Reply:
Whoops, I'd done a search for `Brave` but didn't notice that case sensitive search was on